_posts/2016-04-19_Euklid welcomed as Level39 member.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+external_url: https://www.euklid.com
+membro: euklid
+img: https://www.euklid.com//images/euklid_logo.png
+---
```

Comment:
Mancano le seguenti voci nel "front matter"

title: Euklid al Level 49
bgimg: /img/post-bg.jpg
ruolo: Membro AssoB.it